cylc/flow/scripts/trigger.py

```diff
@@ -27,7 +27,7 @@
 running) at once. They release tasks to run when their active task count
 drops below the queue limit.
 
-Attempts to trigger active tasks will be ignored.
+Attempts to trigger active (submitted, running) tasks will be ignored.
```

Comment:
I think preparing tasks will be ignored too:

```suggestion
Attempts to trigger active (preparing, submitted, running) tasks will be ignored.
```